### PR TITLE
Change the Build name for the Linux portable

### PR DESCRIPTION
**Pull request by jgoshi**
@eerhardt @livarcocc

### CODE REVIEW

**Comment by jgoshi** [comment]
test ci please

File: netci.groovy [hunk 1]
```diff
@@ -19,6 +19,7 @@ def static getBuildJobName(def configuration, def os, def architecture) {
 platformList.each { platform ->
     // Calculate names
     def (os, architecture, configuration) = platform.tokenize(':')
+	def osDisplayName = os;
```
Comment (livarcocc) [inline netci.groovy]: NIT: there is an extra TAB here.

Comment (livarcocc) [inline netci.groovy]: Also, how does this fix it? Isn't osDisplayName always equals to OS? And you were already passing OS before.

**Comment by jgoshi** [comment]
test ci please

**Comment by jgoshi** [comment]
test Debian8.2 x64 Debug Build

**Pull request merged**